action.yaml

```diff
@@ -3,13 +3,20 @@ name: "Datadog JUnitXML Upload"
 description: "Upload JUnitXML reports files to Datadog CI Visibility"
 inputs:
   api-key:
-    required: true
+    required: false
+    description: (Deprecated) Datadog API key to use to upload the junit files.
```

Comment:
We can add [`deprecationMessage`](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddeprecationmessage) param, so that the users could see a warning

Reply:
ah, nice, I'll use that